src/main/java/org/toradocu/conf/Configuration.java

```diff
@@ -146,18 +138,18 @@
 
   @Parameter(
     names = "--aspects-output-dir",
-    description = "Directory where Toradocu will output aspects"
+    description = "Specifies a directory where Toradocu will output aspects"
   )
   private String aspectsOutputDir = "aspects";
 
-  // Javadoc options
-
-  @DynamicParameter(names = "-J", description = "Javadoc options")
-  private Map<String, String> javadocOptions = new HashMap<>();
```

Comment:
How can the user of Toradocu specify options for the Javadoc tool? Why do we need to remove that possibility? Note that this change does not affect much the interface of Toradocu. It simply removes the -J option to pass through options to the Javadoc tool.